nums/numpy/api/stats.py

```diff
@@ -808,7 +808,7 @@ def var(a: BlockArray, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     The doctests shown below are copied from NumPy.
     They won’t show the correct result until you operate ``get()``.
 
-    >>> a = nps.array([[1, 2], [3, 4]]) # doctest: +SKIP
+    >>> a = nps.array([[1, 2], [3, 4]]) # doctest: + SKIP
```

Comment:
I don't know if this matters, but looks like you pushed a minor spacing here.

Reply:
Good catch. Fixed